### ISSUE TITLE
gor does not capture the response

### ISSUE THREAD

**Issue by QasimK** [buger/gor#322]
I'm running the simple gor command `sudo ./gor --input-raw-track-response --input-raw :9001 --output-stdout` and it does not display the responses. Further `sudo ./gor --input-raw-track-response --input-raw :9001 --output-file response.gor` does not save the responses.

It saves neither the response HTTP headers, nor the response body (it saves absolutely nothing regarding the response). It tracks the request just fine. I confirmed this by making two simple POST curl calls sequentially to my local server.

I am using the linux x64 binary from https://github.com/buger/gor/issues/303 containing a fix that is necessary while I trial gor (`Version: DEV-1466094909`).

**Comment by buger** [comment]
Frankly, I can't replicate the issue, tested both Mac and Linux. 

Can you specify which OS are you using?

Also, there were some fixes recently, pls try this binary https://github.com/buger/gor/files/341284/gor_fix_body_size_x64.tar.gz

**Comment by buger** [comment]
Merged #302 into this issue.

**Comment by amirsch** [comment]
Coming from the merged issue (does not return a response), tested both fix_body_size version, and 0.14.1 (on ubuntu 12, and osx el-capitan) on the compiled binaries. 
The response is always 200 OK, in verbose mode we have the proper response though

```
[DEBUG][PID 14110][1469978220087508585][5.675966ms] [HTTPClient] Received: HTTP/1.1 200 OK
Content-Type: text/plain
Date: Sun, 31 Jul 2016 15:17:00 GMT
Connection: keep-alive
Transfer-Encoding: chunked

request successfully proxied!
```

As you can see, the response was supposed to be "request successfully proxied!".
Run with the following command: 

sudo ./gor --input-http :4220 --output-http http://localhost:5852/path -output-http-debug -verbose

**Comment by buger** [comment]
@amirsch you mean it returns only "HTTP/1.1 200 OK" without rest of headers and body? 

Also, your response is not compliant, it has 'Transfer-Encoding: chunked`, but does not have any chunks: https://en.wikipedia.org/wiki/Chunked_transfer_encoding

**Comment by buger** [comment]
@amirsch @QasimK can you try just released  [v0.15.0](https://github.com/buger/gor/releases/tag/v0.15.0) version? Thank you!

**Comment by QasimK** [comment]
Sorry @buger, I won't be able to test for you as I am no longer using gor. Thank you for resolving this issue though (I hope it is at least!)
